bool subscribe
string device_L
string device_R
uint16 width
uint16 height
uint16 fps
uint16 camBufferSize
uint8 compressionRatio
int16 exposure_L
int16 exposure_R
string calibrationFile
---
int8 ret
string errorDesc